src/Repositories/TestRepository.php

```diff
@@ -104,7 +104,14 @@ public function use(array $classOrTraits, array $groups, array $paths): void
         }
 
         foreach ($paths as $path) {
-            $this->uses[$path] = [$classOrTraits, $groups];
+            if (array_key_exists($path, $this->uses)) {
+                $this->uses[$path] = [
+                    array_merge($this->uses[$path][0], $classOrTraits),
+                    array_merge($this->uses[$path][1], $groups)
```

Comment:
This is missing a trailing comma (`trailing_comma_in_multiline_array`). 👍
```suggestion
                    array_merge($this->uses[$path][1], $groups),
```